### ISSUE TITLE
[white Label - Audiocard]: Narrator focus is moving to hidden after No buttons,… text while navigating using arrow keys.

### ISSUE THREAD

**Issue by Kusum199** [microsoft/BotFramework-WebChat#5714]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Screen reader users hear misleading information about the element being part of a list even when it is not. This creates confusion, disrupts navigation, and makes it harder for users to understand the actual structure and purpose of the audio file element.

### What browsers and screen readers do this issue affect?

Others, Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### How to reproduce the issue?

Pre-Requisite:
Turn on Narrator

Repro Steps
1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3.Navigate to white label experience chat page and type audiocard word in type your message text box.
4. Response will appear.
5. Navigate to the Audio file using arrow keys.
6. Observe an issue that Narrator announcing unnecessary list information when focus lands on whole audio file while navigating using arrow keys.

### What do you expect?

The element should expose only the appropriate programmatic role and semantics for an audio file. Narrator should announce only relevant information about the audio file and should not include list‑related semantics unless the element is explicitly a list or listitem.

### What actually happened?

Narrator focus is moving to hidden after No buttons,… text while navigating using arrow keys.

### Do you have any screenshots or recordings to repro the issue?

https://github.com/user-attachments/assets/1ad84d9a-ae28-4c4f-8914-7a9150ac022a

### Adaptive Card JSON

```json

```
